Tests/L0/Maven/_suite.ts

```diff
@@ -1491,4 +1617,161 @@ describe('Maven Suite', function () {
                 done(err);
             });
     })
+
+    /* Unit tests - SonarQube common code */
```

Comment:
New tests operate at a method-by-method level rather than running the task and expecting certain outputs. This is much more fine-grained and offers opportunities for better code coverage.